tests/e2e/registration-by-sms.js

```diff
@@ -3,6 +3,20 @@ const utils = require('../utils'),
       helper = require('../helper'),
       moment = require('moment');
 
+const computeExpectedDate = async () => {
+  const reportedDate = await browser.executeAsyncScript(async () => {
+    const callback = arguments[arguments.length - 1];
+    const element = document.querySelector('.detail .relative-date-content');
```

Comment:
I think this selector should be more specific - it would be easy to add another date to the page which would cause this to fail unnecessarily.